### PR TITLE
perf: improve paragragh performance

### PR DESCRIPTION
**Pull request by rfyiamcool**
### summary

improve paragragh performance.

#### before optimize

```
goos: darwin
goarch: amd64
pkg: github.com/go-faker/faker/v4
cpu: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
BenchmarkParagraph
BenchmarkParagraph-12    	   21318	     50666 ns/op	   20836 B/op	     164 allocs/op
PASS
ok  	github.com/go-faker/faker/v4	1.833s
```

#### after optimize

```
goos: darwin
goarch: amd64
pkg: github.com/go-faker/faker/v4
cpu: Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
BenchmarkParagraph
BenchmarkParagraph-12    	   25045	     43137 ns/op	   17784 B/op	      94 allocs/op
PASS
ok  	github.com/go-faker/faker/v4	1.867s
```

### CODE REVIEW

File: lorem_test.go [hunk 1]
```diff
@@ -52,13 +52,15 @@ func TestFakeWord(t *testing.T) {
 
 func TestFakeSentence(t *testing.T) {
 	res := Sentence()
+	fmt.Println(res)
```
Comment (bxcodec) [inline lorem_test.go:55]: can remove this print please

File: lorem_test.go [hunk 2]
```diff
@@ -52,13 +52,15 @@ func TestFakeWord(t *testing.T) {
...
 	if res == "" || !strings.HasSuffix(res, ".") {
 		t.Error("Expected sentence")
 	}
 }
 
 func TestFakeParagraph(t *testing.T) {
 	res := Paragraph()
+	fmt.Println(res)
```
Comment (bxcodec) [inline lorem_test.go:63]: can remove this please?

File: lorem_test.go (hunk 1 above)
Comment (rfyiamcool) [reply lorem_test.go]: done

File: lorem_test.go (hunk 2 above)
Comment (rfyiamcool) [reply lorem_test.go]: During the test, the results were printed to see if they were as expected. When I submitted the PR, I forgot to delete it. 😆

Comment (rfyiamcool) [reply lorem_test.go]: > can remove this please?

done

**Review by bxcodec — APPROVED** [review]
LGTM

File: lorem_test.go (hunk 2 above)
Comment (ionling) [reply lorem_test.go]: I like to use `t.Log()` here.

**Pull request merged**